spec/_attachments/ic.did

```diff
@@ -3,6 +3,7 @@ type wasm_module = blob;
 
 type log_visibility = variant {
     controllers;
+    allowlist : opt vec principal;
```

Comment:
Why is the vector optional? What is the semantics if no list of principals is provided?

Reply:
that was added by mistake. thanks for pointing this out. removed.